### PR TITLE
Add some more docs about how windows are configured and what to expect from them.

### PR DESCRIPTION
**Pull request by mfelsche**

### CODE REVIEW

File: docs/tremor-query/index.md [hunk 1]
```diff
@@ -113,14 +113,49 @@ A tumbling window is a window configured with a fixed non-overlapping interval o
...
+
+- `size`: Number of events until this window closes and emits a downstream event (The ).
+
+The size increment for each event defaults to `1` but can be customized by the embedded script in the window definition.
+This script needs to return an unsigned integer denoting the number of events to use for this event.
+It is possible to ignore the current event by emitting `0`.
+
+#### Windows based on time
+
+Time based tumbling windows close when a certain duration has been elapsed. The source for measuring the duration
```
Comment (darach) [inline docs/tremor-query/index.md:138]: has been elapsed -> has elapsed

File: docs/tremor-query/index.md [hunk 2]
```diff
@@ -113,14 +113,49 @@ A tumbling window is a window configured with a fixed non-overlapping interval o
...
+is the `ingest` timestamp of the events flowing through by default. The provided embedded script can be used to customize the
+source of time measurement. The embedded script must return a number representing a timestamp in nanoseconds. 
+This way windows using other timestamps than event ingest time can be built.
```
Comment (darach) [inline docs/tremor-query/index.md:141]: than -> that

Comment (mfelsche) [reply docs/tremor-query/index.md:141]: really? `other timestamps that event ingest time`?

File: docs/tremor-query/index.md [hunk 3]
```diff
@@ -113,14 +113,49 @@ A tumbling window is a window configured with a fixed non-overlapping interval o
...
+
+* `eviction_period`: duration in nanoseconds without events arriving, after which to close an existing window, if it is full or not.
+
+#### Windows based on number of events
+
+Size based tumbling windows close when a certain number of aggregated events has been reached.
+
+Configuration Parameters:
+
+- `size`: Number of events until this window closes and emits a downstream event (The ).
```
Comment (Licenser) [inline docs/tremor-query/index.md:130]: `(The )` seems to be missing something

File: docs/tremor-query/index.md (hunk 2 above)
Comment (Licenser) [reply docs/tremor-query/index.md:141]: `than` reads better to me too, I never heard `other that`

perhaps rephrasingf would make ti cleaner?
```
This way windows using timestamps other than the event ingest time can be built.
```

**Review by darach — APPROVED** [review]
LGTM 👍

**Review by Licenser — APPROVED** [review]

**Pull request merged**